styles.css

```diff
@@ -1,5 +1,10 @@
 body {
   margin: 0;
+  font-family: 'Crete Round', sans-serif;
```

Comment:
- [ ] Thank you for changing the <code>font-family</code> as requested by the previous reviewer 🥰  but note that the <code>'Crete Round'</code> only applied to the **title**. For the rest it is <code>Inter</code> 🤓🤓